### ISSUE TITLE
Repeated method name "getAllCities" in Controllers

### ISSUE THREAD

**Issue by rodrigojv** [FundacionParaguaya/FP-PSP-SERVER#36]
There are some controllers that have the name `getAllCities` and this name does not reflect what the method is returning. For example in `UserRoleController`.

As it is now:
```java
public class UserRoleController {

	@GetMapping()
	public ResponseEntity<List<UserRoleDTO>> getAllCities() {
		List<UserRoleDTO> userRoles = userRoleService.getAllUserRoles();
		return ResponseEntity.ok(userRoles);
	}
```

Preferred way:
```java
public class UserRoleController {

	@GetMapping()
	public ResponseEntity<List<UserRoleDTO>> getAllUserRoles() {
		List<UserRoleDTO> userRoles = userRoleService.getAllUserRoles();
		return ResponseEntity.ok(userRoles);
	}
```

**Comment by gzacur** [comment]
@mcespedeso please review this feedback :)

**Comment by mcespedeso** [comment]
All right ! thanks for the suggestion of change

**Comment by rodrigojv** [comment]
Closed by #37.
